modin/pandas/test/test_io.py

```diff
@@ -2613,6 +2629,56 @@ def test_to_pickle(self):
             assert assert_files_eq(unique_filename_modin, unique_filename_pandas)
 
 
+class TestXml:
+    def test_read_xml(self):
+        # example from pandas
+        data = """<?xml version='1.0' encoding='utf-8'?>
+<data xmlns="http://example.com">
+ <row>
+   <shape>square</shape>
+   <degrees>360</degrees>
+   <sides>4.0</sides>
+ </row>
+ <row>
+   <shape>circle</shape>
+   <degrees>360</degrees>
+   <sides/>
+ </row>
+"""
+        eval_io("read_xml", path_or_buffer=data)
+
+
+class TestOrc:
```

Comment:
For some formats (`orc`, `spss`), it is not easy to add infrastructure. In cases where we default to pandas, I think it's enough to check that the parameters are passed to pandas.

Reply:
this should be in-code comment 😉

Reply:
added